doc/source/about/index.rst

```diff
@@ -30,36 +30,30 @@ to the project.  For a list of those members and a description of their
 contributions to the code, see
 `our members website. <https://yt-project.org/members.html>`_
 
-For an up-to-date list of everyone who has contributed to the yt codebase,
-see the current `CREDITS <https://github.com/yt-project/yt/blob/main/CREDITS>`_ file.
-For a more detailed breakup of contributions made by individual users, see out
-`Open HUB page <https://www.openhub.net/p/yt_amr/contributors?query=&sort=commits>`_.
-
 History of yt
 -------------
 
-yt was originally begun by Matthew Turk in 2007 in the course of his graduate
-studies in computational astrophysics.  The code was developed
-as a simple data-reader and exporter for grid-based hydrodynamical simulation
-data outputs from the *Enzo* code.  Over the next few years, he invited
-collaborators and friends to contribute and use yt.  As the community grew,
-so did the capabilities of yt.  It is now a community-developed project with
-contributions from many people, the hospitality of several institutions, and
-benefiting from numerous grants.  With this community-driven approach
-and contributions from a sizeable population of developers, it has evolved
-into a fully-featured toolkit for analysis and visualization of
-multidimensional data.  It relies on no proprietary software -- although it
-can be and has been extended to interface with proprietary software and
-libraries -- and has been designed from the ground up to enable users to be
-as immersed in the data as they desire.
+yt was originally created to study datasets generated by cosmological
+simulations of galaxy and star formation conducted by the simulation code Enzo.
```

Comment:
Sounds a little off to me but maybe that's how people actually phrase it ?
```suggestion
simulations of galaxies and star formation conducted by the simulation code Enzo.
```

Reply:
I think I'm going to leave this but I'll wait to hear from @chummels

Reply:
Ok 👍